source/includes/_tipser-script.md

```diff
@@ -688,3 +687,42 @@ If you'd like to change other elements' color as well, please use specific class
 A working examples of page based on Tipser Widget can be found on <a href="https://tipser.github.io/tipser-widget-bootstrap/" target="_blank">Tipser Widget Bootstrap page</a>.
 
 The code of that page is available as a GitHub <a href="https://github.com/Tipser/tipser-widget-bootstrap" target="_blank">Tipser Widget Bootstrap project</a>. Feel free to checkout it and play with it on your local machine!
+
+## Advanced usages of Script
+
+If your use of Tipser Script is more complex than a default setup, you can use some of our more advanced options allowing you to control the checkout process. 
+
+### advanced Checkout functions
+
+Even if you don't use React in your project, we allow you to access the checkout context and create custom functions for submitting delivery and billing address forms as well as Stripe payment details. 
+
+To access the context, create an Event Listener attached to your checkout wrapper (the element with `data-tipser-modular-checkout` attribute):
+
+```javascript
+const modularCheckoutDiv = document.querySelector('#modular_checkout_root');
+let getCheckoutContext = null;
+
+modularCheckoutDiv.addEventListener('checkout-context-ready', (evt) => {
+  getCheckoutContext = evt.detail.getCheckoutContext;
+});
+```
+
+Now create your custom Submit button, eg: 
+
+```html
+<button id="submit-button" onclick="customSubmitDeliveryAddress()"></button>
+```
+
+and define your submit delivery address function:
+
+```javascript
+const customSubmitDeliveryAddress = () => {
```

Comment:
Dziwnie że ta funkcja się nazywa `customSubmitDeliveryAddress ` a submituje delivery, billing oraz payment ;-) . Ale może to jest clear enough.